workbench-core/authentication/src/authenticationPlugin.ts

```diff
@@ -70,14 +70,15 @@ export interface AuthenticationPlugin {
    *
    * @param code - an authorization code
    * @param codeVerifier - the PKCE code verifier
+   * @param websiteUrl - the url to redirect to after login is completed. Must be the same url used in the {@link getAuthorizationCodeUrl} function
```

Comment:
would it be more accurate for this to be named redirectUrl?

Reply:
my thinking with `websiteUrl` is that its more obvious that this is the url of your frontend UI